curp/src/cmd.rs

```diff
@@ -45,7 +43,7 @@ pub trait Command:
 
     /// Execute the command after_sync callback
     #[inline]
-    async fn after_sync<E>(&self, e: &E, index: LogIndex) -> Result<Self::ASR, E::Error>
+    async fn after_sync<E>(&self, e: &E, index: usize) -> Result<Self::ASR, E::Error>
```

Comment:
Why do we use `usize` instead of `LogIndex`?

Reply:
discussed above